packages/docs/src/styles/components/_ReactPropDocs.scss

```diff
@@ -0,0 +1,50 @@
+// Responsive docs table
+
+.docs_table--wrapper {
+  overflow-x: auto;
+}
+
+.docs_table {
+  @media (max-width: $width-sm) {
+    thead {
+      /* Hide table headers (.ds-u-visibility--screen-reader) */
+      border: 0;
+      clip: rect(0, 0, 0, 0);
+      height: 1px;
+      overflow: hidden;
+      padding: 0;
+      position: absolute;
+      width: 1px;
+      // Workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1241631
+      word-wrap: normal;
+    }
+
+    tr {
+      border: 1px solid $color-gray;
+      display: block;
+      margin: 0 0 0.5rem;
+    }
+
+    td {
+      border-top: 1px solid $color-gray-lightest;
+      display: block;
+      margin-top: -1px;
```

Comment:
I think you can remove the negative margin on `margin-top` and add a line above `border-top` for `border: none;` This gets rid of the negative margin needed and implements the borders a bit cleaner.

Reply:
Thanks, that does look cleaner.